pkg/util/testing/wrappers.go

```diff
@@ -464,6 +464,22 @@ func (p *PodSetWrapper) PodSetGroup(name string) *PodSetWrapper {
 	return p
 }
 
+func (p *PodSetWrapper) PodSetSliceRequiredTopology(level string) *PodSetWrapper {
```

Comment:
```suggestion
func (p *PodSetWrapper) SliceRequiredTopologyRequest(level string) *PodSetWrapper {
```

This function is only for PodSetWrapper which means this function has already `PodSet` semantic similar to `PreferredTopologyRequest() (Line 483).

Reply:
Done